frontend/src/containers/AdminPanel/AdminPanel.js

```diff
@@ -68,6 +70,11 @@ class AdminPanel extends Component {
   render() {
     const { user, setJwt } = this.props;
     const { activeTab, recipes, users } = this.state;
+    const isAdmin = isNil(user) ? 'loadingPage' : user.is_admin;
```

Comment:
I don't think `isAdmin` should be responsible for determining whether the pace is loading, since that's a pretty different concept.